scripts/binary/binary-cleanup.js

```diff
@@ -114,13 +113,53 @@ const getDependencyPathsToKeep = async (buildAppDir) => {
   return [...Object.keys(esbuildResult.metafile.inputs), ...entryPoints]
 }
 
-const cleanup = async (buildAppDir) => {
+const createServerEntryPointBundle = async (buildAppDir) => {
+  const unixBuildAppDir = buildAppDir.split(path.sep).join(path.posix.sep)
+  const entryPoints = [path.join(unixBuildAppDir, 'packages/server/index.js')]
```

Comment:
Should this be:
```suggestion
  const entryPoints = [unixBuildAppDir, 'packages', 'server' , 'index.js'].join(path.posix.sep)
```

Reply:
I am trying to limit node internals references as much as I possibly can. `path.posix.sep` is something we already handle uniquely in the snapshot (since when we're generating the snapshot we don't have access to `path` or any other node internals), so I just decided to reuse what we had available.

See: 

https://github.com/cypress-io/cypress/blob/develop/tooling/v8-snapshot/src/generator/blueprint.ts#L56

and

https://github.com/cypress-io/cypress/blob/develop/tooling/v8-snapshot/src/generator/blueprint.ts#L110